Update to 1.16.5

AdanethMirime opened the pull request:
Update to 1.16.5

AdanethMirime committed:
Update to 1.16.5

Update to 1.16.5

pascallj reviewed (approved):
Looking good, thanks! 👍 

Will be merged once I have tested it.

_EDIT: Sorry I approved a bit too soon. New review below._

pascallj commented on src/main/resources/META-INF/mods.toml:
```diff
@@ -40,6 +40,6 @@ loaderVersion="[29,)" #mandatory This is typically bumped every Minecraft versio
 		    modId="minecraft"
 		    mandatory=true
-		    versionRange="[1.15,1.15.2]"
+		    versionRange="[1.16,1.17)"
```
Please remove 1.17 here, as it is not released yet. I don't like assuming future versions will automagically work.

AdanethMirime replied:
Sure, I've changed it to `[1.16,1.16.5]`.

pascallj reviewed (changes requested):
Apart from one minor change, looking good, thanks! :+1: 

I will still have to test though.

AdanethMirime committed:
Fix upper limit of version & add info to readme

Upper limit of Minecraft version changed to 1.16.5.  Also made a slight change to the readme adding information about incremental rotation versus smooth.

pascallj commented:
Thank you, working fine!
